src/style.css

```diff
@@ -0,0 +1,115 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 10px;
+  line-height: 14px;
+}
+    
+body {
+  margin: 0;
+}
+  
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-left: 16px;
+  padding-bottom: 16px;
+}
+    
+.card__description {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  padding-right: 16px;
+  color: #060b35;
+}
+    
+.card__logo {
+  display: flex;
+  justify-content: center;
+  width: 160px;
+  padding-top: 32px;
+  padding-bottom: 40px;
+}
+    
+.card__link {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+  width: 166px;
+  height: 40px;
+  color: white;
+  text-decoration: none;
+  background: #00acdc;
+  bottom: 17px;
+  border-radius: 5px;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+}
+    
+.card__link:hover {
+  background: #fff;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
```

Comment:
Use CSS variable for repeatable colors.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)